diff --git a/Makefile b/Makefile
index ad13dd1..cbbe7bc 100644
--- a/Makefile
+++ b/Makefile
@@ -3,6 +3,7 @@ MAN=$(PREFIX)/man/man8
 SBIN=$(PREFIX)/sbin
 CONF=$(PREFIX)/etc/node-agent.d
 MODULES=$(PREFIX)/etc/node_modules
+SMF=$(PREFIX)/lib/svc/manifest/network/circonus
 MAKE?=make
 
 all:
@@ -29,8 +30,8 @@ install-modules:
 
 install-illumos:	install
 	/bin/sed -e "s#@@PREFIX@@#$(PREFIX)#g" smf/nad.xml.in > smf/nad.xml
-	mkdir -p $(DESTDIR)/lib/svc/manifest/network/circonus
-	./install-sh -c -m 0644 smf/nad.xml $(DESTDIR)/lib/svc/manifest/network/circonus/nad.xml
+	mkdir -p $(DESTDIR)$(SMF)
+	./install-sh -c -m 0644 smf/nad.xml $(DESTDIR)$(SMF)/nad.xml
 	cd $(DESTDIR)$(CONF)/illumos ; $(MAKE)
 	cd $(DESTDIR)$(CONF) ; for f in aggcpu.elf cpu.elf fs.elf if.sh sdinfo.sh smf.sh tcp.sh vminfo.sh vnic.sh zfsinfo.sh zone_vfs.sh; do /bin/ln -sf illumos/$$f ; done
 
